src/components/plots/PlotsTablesContainer.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Space, Divider, Card, Row, Col,
+} from 'antd';
+import Link from 'next/link';
+
+import { plotNames, layout } from 'utils/constants';
+import heatmap from '../../../public/static/media/heatmap.png';
+import embeddingContinuous from '../../../public/static/media/embeddingContinuous.png';
+import embeddingCategorical from '../../../public/static/media/embeddingCategorical.png';
+import violin from '../../../public/static/media/violin.png';
+import dotPlot from '../../../public/static/media/dotplot.png';
+import volcano from '../../../public/static/media/volcano.png';
+import frequency from '../../../public/static/media/frequency.png';
+import markerHeatmap from '../../../public/static/media/marker_heatmap.png';
```

Comment:
change paths to absolute?

Reply:
These can't be changed to the absolute path because they lay outside the `src` folder. We'd have to add a new path for these files. I don't think it's worth it because we don't use it a lot, so I'd leave this.